moveit_ros/visualization/src/moveit_ros_visualization/moveitjoy_module.py

```diff
@@ -348,7 +348,7 @@ def new(self, status, attr):
 
 class MoveitJoy:
     def parseSRDF(self):
-        ri = RobotInterface("/robot_description")
+        ri = RobotInterface("/robot_description", "")
```

Comment:
@willcbaker #842 shows a similar issue like the one fixed here.
Why did you added an empty namespace here, instead of using an empty default in the `RobotInterface` constructor?